app/assets/javascripts/jquery/plugins/inat/taxon_selectors.js.erb

```diff
@@ -97,6 +97,7 @@
     var status = $('<div class="status">' + I18n.t('species_unknown') + '</div>');
     $(status).css(
       $.extend(
+        {},
         $.fn.simpleTaxonSelector.styles.statuses['default'],
         $.fn.simpleTaxonSelector.styles.statuses.unmatched
       )
```

Comment:
In case you don't already know, you can do this instead without needing lodash/underscore:

```js
$(status).css({
    ...$.fn.simpleTaxonSelector.styles.statuses['default'],
    ...$.fn.simpleTaxonSelector.styles.statuses.unmatched
})
```

Reply:
Thanks for reaching out. There's quite a lot that could be improved with modern JS here, but I'm trying to stay consistent within a code unit, and to make minimal changes. Also, the most important calls to `extend` here perform deep merges, which I don't think I can do with plain JS.